lib/Parse/ParseStmt.cpp

```diff
@@ -2529,3 +2548,28 @@ ParserResult<Stmt> Parser::parseStmtPoundAssert() {
   return makeParserResult<Stmt>(new (Context) PoundAssertStmt(
       SourceRange(startLoc, endLoc), conditionExprResult.get(), message));
 }
+
+Optional<tok> Parser::closestMatchingStmtTokenForIdentifier(StringRef identifier) {
```

Comment:
Do we need to make it a Parser method? If not, please make this a `static` function.

Reply:
Sure, fixed